astropy/io/fits/scripts/fitsdiff.py

```diff
@@ -292,7 +292,7 @@ def main():
     if opts.quiet:
         out_file = None
     elif opts.output_file:
-        out_file = open(opts.output_file, 'wb')
+        out_file = open(opts.output_file, 'w')
```

Comment:
Any specific reason for this change?

Reply:
Yes, it was not working :wink: (writing unicode strings to a file in binary mode). 
Actually I tried to remove this as `diff.report` supports writing to a filename, but the same file is also used to capture the logging output and this was causing conflicts. All this is clearly not optimal, but the goal here is to add tests, I refrained myself to go further!

Reply:
okay, that makes sense. Weird design there ...